lang/id/index.md

```diff
@@ -0,0 +1,164 @@
+dasar untuk menentukan bagaimana cara menentukan versi, dan bagaimana cara menambah angka-angka
+di versi tersebut. Aturan ini hanyalah dasar, dan tidak untuk membatasi standar versi yang 
+sebelumnya sudah digunakan.
+
+Supaya standar ini bisa bermanfaat, pertama kalian harus menentukan API publik, titik
+dimana kalian mulai menggunakan standar ini. Setelah ditentukan, setiap rilis
+versi bisa kalian komunikasikan dengan dokumentasi, dan secara langsung melalui angka versi
+tersebut. Pembetulan bug menambah angka patch, penambahan fitur menambah angka minor, perubahan
+yang membuat versi lama tidak bisa digunakan menambah angka major.
+
+Standar ini bernama "Versi Semantik", dengan standar ini, setiap orang yang melihat
+angka versi bisa tahu secara umum apa yang berubah dengan sistem tersebut.
+
+Spesifikasi Versi Semantik (VerSem)
+------------------------------------------
+
+Kata HARUS, TIDAK BOLEH, DIBUTUHKAN, SEHARUSNYA, JANGAN SAMPAI, SEBAIKNYA, SEBAIKNYA TIDAK,
+DIREKOMENDASIKAN, BISA, dan OPSIONAL di dokumen ini sesuai dengan [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+
+1. Perangkat lunak dengan Versi Semantik harus menentukan API public. Bisa
+dijelaskan dengan kode, atau ditulis di dokumentasi, ditulis dengan jelas dan akurat.
+
+2. Versi HARUS ditulis dalam bentuk X.Y.Z, dengan X, Y, Z angka bulat positif, dan TIDAK
+BOLEH didahului angka 0 (contoh 01.02.03). X adalah versi MAJOR, Y adalah minor, dan Z adalah patch.
+
+3. Setelah versi dirilis, isi dari versi tersebut TIDAK BOLEH dirubah. Setiap perubahan harus
+merilis versi baru.
+
+4. Versi major 0 (0.y.z) adalah untuk pengembangan, dan bisa terjadi banyak perubahan. API publik
+dianggap tidak stabil di versi ini.
+
+5. Versi 1.0.0 adalah titik awal API publik, setiap versi baru ditulis berdasarkan versi ini.
+
+6. Versi patch Z (x.y.Z | x > 0) HARUS ditambah jika ada perbaikan bug tanpa menambah fitur.
+
+7. Versi minor Y (x.Y.z | x > 0) HARUS ditambah jika ada fitur baru, atau ada fitur lama
+yang ditandai usang, bisa dirubah bersama dengan versi patch, jika minor ditambahkan, maka versi patch harus dikembalikan ke angka 0.
+
+8. Versi major X (X.y.z | X > 0) HARUS ditambah jika ada perubahan yang membuat versi baru
+tidak kompatibel dengan versi lama, seperti menghapus fitur lama, menambah fitur baru yang tidak bisa
+digunakan di versi lama, bisa dirubah bersama dengan versi patch dan minor, jika major ditambahkan, maka versi minor dan patch harus dikembalikan ke angka 0.
+
+9. Versi sebelum rilis BISA ditulis dengan menambahkan garis dan
+bisa dipisah dengan titik tepat setelah versi patch. Versi sebelum rilis HARUS ditulis dengan
+huruf ASCII alfanumerik dan garis [0-9A-Za-z], TIDAK BOLEH kosong, dan angka TIDAK BOLEH
+didahului dengan angka 0. Versi sebelum rilis dianggap tidak stabil dan dikesampingkan jika ada versi yang stabil. Contoh: 1.0.0-alpha, 2.3.1-beta.
+
+10. Data tambahan BISA ditulis setelah versi sebelum rilis didahului dengan tanda tambah
```

Comment:
I am not sure if there's a translation for Build Metadata, but it's not Data Tambahan